src/packing_queue/tables/HistoryTable.jsx

```diff
@@ -325,38 +325,40 @@ const HistoryTable = ({
     }, 0);
   };
 
-  const onDownloadPDFClick = useCallback(async () => {
-    await API.downloadPDF(
-      selectedRow._id,
-      selectedRow.orderNumber,
-      selectedRow.dateCreatedValue
-    )
-      .then((data) => {
-        pdfMake.createPdf(data.docDefinition).open();
-        setMenuPosition(null);
-        enqueueSnackbar("Packing slip downloaded", snackbarVariants.error);
-      })
-      .catch((e) => {
-        enqueueSnackbar(e.message, snackbarVariants.error);
-      });
-  }, [selectedRow, enqueueSnackbar]);
+  //No longer used mm - 081522
+  // const onDownloadPDFClick = useCallback(async () => {
+  //   await API.downloadPDF(
+  //     selectedRow._id,
+  //     selectedRow.orderNumber,
+  //     selectedRow.dateCreatedValue
+  //   )
+  //     .then((data) => {
+  //       pdfMake.createPdf(data.docDefinition).open();
+  //       setMenuPosition(null);
+  //       enqueueSnackbar("Packing slip downloaded", snackbarVariants.error);
+  //     })
+  //     .catch((e) => {
+  //       enqueueSnackbar(e.message, snackbarVariants.error);
+  //     });
+  // }, [selectedRow, enqueueSnackbar]);
 
   const historyRowMenuOptions = useMemo(
     () => [
       <MenuItem key={"View"} onClick={openViewPackingSlip}>
         View
       </MenuItem>,
-      <MenuItem key={"Download"} onClick={onDownloadPDFClick}>
-        Download
-      </MenuItem>,
+      //No longer used mm - 081522
+      // <MenuItem key={"Download"} onClick={onDownloadPDFClick}>
+      //   Download
+      // </MenuItem>,
       <MenuItem key={"Edit"} onClick={openEditPackingSlip}>
         Edit
       </MenuItem>,
       <MenuItem key={"Delete"} onClick={openDeleteDialog}>
         Delete
       </MenuItem>,
     ],
-    [onDownloadPDFClick]
+    // [onDownloadPDFClick]
```

Comment:
You'll need to keep the empty array here so that `historyRowMenuOptions` is not recomputed every render. An empty array will allow it to be computed once on the initial mount.

Reply:
Added in empty array.